tests/conftest.py

```diff
@@ -147,18 +149,18 @@ def request(
         body: dict = None,
         headers: dict = None,
         timeout: typing.Union[TimeoutTypes, UseClientDefault] = USE_CLIENT_DEFAULT,
-    ) -> tuple:
+    ) -> httpx.Response:
         self.request_calls.append(RequestArgs(url, method, body, headers, timeout))
         return super().request(url, method, body, headers=headers, timeout=timeout)
 
-
 @pytest.fixture
-def client():
-    url = os.getenv("SCHEMA_REGISTRY_URL", "http://localhost:8081")
-    client = RequestLoggingSchemaRegistryClient(url)
-    yield client
-
-    subjects = {
+def test_subjects() -> typing.List[str]:
```

Comment:
the subjects set for both the client and async_client fixtures got out of date. So I moved the them to a separate fixture, so future updates happen in just one place.